tests/Microsoft.Identity.Test.Unit/CoreTests/RegionDiscoveryProviderTests.cs

```diff
@@ -127,7 +127,7 @@ private void AddMockedResponse(HttpResponseMessage responseMessage)
                     new MockHttpMessageHandler
                     {
                         ExpectedMethod = HttpMethod.Get,
-                        ExpectedUrl = "http://169.254.169.254/metadata/instance/compute/api-version=2020-06-01",
+                        ExpectedUrl = "http://169.254.169.254/metadata/instance/compute",
```

Comment:
You have some tests failing which are easy to fix, just a small note to rename them `happyPathAsync` to a more meaningul name as it's hard to understand which test failed when lookinag the CI system. Also, please try to use PascalCase for all methods, private and public. I'll see about updating our code style to emit warnings.

Reply:
Makes sense. Updated the names.